src/ui/HelpPage.tsx

```diff
@@ -0,0 +1,21 @@
+import * as React from 'react';
+import { render } from 'react-dom';
+
+
+class HelpPage extends React.Component {
+public render() {
+  return (
+    <div>
+      <h1>Virtuaalimonitorin käyttöopas</h1>
+        <h2>Virtuaalimonitorin käyttö selainparametrien avulla </h2>
+       <p>
+         Pysäkkinäytön käyttö selaimen osoiteriviltä tapahtuu seuraavasti: kirjoita osoitteen perään /stop/pysäkit/rivimäärä. Esimerkiksi /stop/tampere:0010/10 näyttää 10 riviä pysäkiltä Keskustori F. 
```

Comment:
Maybe we should give instructions for where you can get the stop in this format. Maybe the easiest way would be to go to stop's page on digitransit-ui and you will see it in the URL. However, then we should also list the available addresses for the digitransit-ui instances that are running for the available digitransit-virtualmonitor configurations

Reply:
/stop/pysäkki/rivimäärä*

Reply:
I think that the pysäkit term is still more correct here. I'm using a stop in that example thats true, but I think it would be misleading if we put only pysäkki term, since you can use it with one or multiple stops. Also something like 'pysäkki/pysäkit' would be more misleading in my opinion.

Reply:
> Maybe we should give instructions for where you can get the stop in this format. Maybe the easiest way would be to go to stop's page on digitransit-ui and you will see it in the URL. However, then we should also list the available addresses for the digitransit-ui instances that are running for the available digitransit-virtualmonitor configurations

Or would it be more accurate if we make the help page's information to correspond current instance we are using. For example, tampere if an user comes from tre,.. other if user comes from elsewhere